swagger.yaml

```diff
@@ -1355,6 +1355,221 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/articles":
+    get:
+      operationId: getArticles
+      summary: List articles
+      description: Get a list of articles
+      tags:
+        - Article
+      parameters:
+        - name: path
+          required: false
+          in: query
+          description: Get articles with specified path.
+          schema:
+            type: string
+        - name: draft
+          required: false
+          in: query
+          description: Get draft articles.
+          schema:
+            type: boolean
+      responses:
+        "200":
+          description: List of articles.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ArticleMetadata"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                  $ref: "#/components/schemas/Error"
+
+    post: 
+      operationId: createArticle
+      summary: Post a new article
+      description: Add new article
+      tags:
+        - Article
+      requestBody:
+        description: New article entry
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Article"
+      responses:
+        "200":
+          description: Created new article entry
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Article"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  "/articles/{path}":
```

Comment:
Find endpoints should not accept path. Instead they should use ID, so instead of doing this you should have
list endpoint with query parameter path:
    /articles?path=something
and find endpoint with:
    /articles/{articleId}